### PR TITLE
Remove f2py code and restructure for pypi

### PR DESCRIPTION
**Pull request by arporter**
This pull request is a result of work under #1 on the pypi_restructure branch. I have removed all un-needed code relating to f2py, keeping only the fparser and Fortran2003 parsing code.

### CODE REVIEW

**Comment by arporter** [comment]
I've done as suggested. I don't like it very much as we have almost the same thing twice but it does seem to be the safest option. I've added the new license text to all src files because they've been modified in the sense that the tree structure has been changed and we will probably modify them further in future.

The only files I've not added the license to are those that are documentation related (e.g. README.md) because I couldn't find a way of doing that without having the license text appear in the generated document.

**Comment by arporter** [comment]
Contrary to what I just said, it is only mark-down files that I don't seem to be able to add the license text to. I've just been through and updated all others.

Also, forgot to say that yes, the link to code metrics in the README are for the master branch.

Time for a 'second review' (or whatever the github term for that is)...

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,43 @@
-Copyright (c) 1999-2008 Pearu Peterson
+Modified work Copyright (c) 2017 Science and Technology Facilities Council
+Original work Copyright (c) 1999-2008 Pearu Peterson
 
 All rights reserved.
 
+Modifications made as part of the fparser project are distributed
+under the follosing license:
```
Comment (rupertford) [inline LICENSE]: typo: should be following

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2017 Science and Technology Facilities Council
+
+# All rights reserved.
+
+# Modifications made as part of the fparser project are distributed
+# under the follosing license:
```
Comment (rupertford) [inline .gitignore]: typo: following, not follosing

**Comment by arporter** [comment]
Arrgh. I thought after I'd done it that I hadn't checked for typos. Anyway, I've fixed that now.
`git diff master | grep follosing` gives nothing.

**Pull request merged**